server/protocol.c

```diff
@@ -681,245 +681,102 @@ static int field_name_len(const char *field)
     return end - field;
 }
 
-static const char m_invalid_str[] = "-";
-
-static int read_request_line(request_rec *r, apr_bucket_brigade *bb)
+typedef enum {
+    ra_none,
+    ra_badprotocol,
+    ra_reject09,
+    ra_badmethod09,
+} ra_error;
```

Comment:
`ra_`?? Maybe rl_error_ (for request_line) or http_error_?

Reply:
Changed to `http_error` prefix.